core/store/migrate/migrations/0090_ocr2_relay.sql

```diff
@@ -0,0 +1,28 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE offchainreporting2_oracle_specs
+    ADD COLUMN relay text,
+    ADD COLUMN relay_config JSONB,
+    DROP COLUMN evm_chain_id,
+    DROP CONSTRAINT chk_contract_address_length;
+ALTER TABLE offchainreporting2_oracle_specs
+    RENAME COLUMN contract_address TO contract_id;
```

Comment:
Is this still a `bytea`? Should it be a `text` now?

Reply:
yeah it still works with bytea, but probably cleaner as text. Updated, however we can't migrate back down bytea but thats ok